incident-response/overview.md

```diff
@@ -0,0 +1,198 @@
+Command System (ICS)](https://en.wikipedia.org/wiki/Incident_Command_System).
+This document will describe how we adopt this process to work for our
+projects.
+
+## What Is An Incident
+
+Before we describe incident response, we should define what exactly
+constitutes an incident. An incident is a service disruption, usually
+with user impact, that needs to be addressed outside of the normal
+sprint process, and requires an increased degree of coordination and
+communication.
+
+This definition is intentionally vague because there's no hard and fast
+rule about what qualifies for the incident response process. Basically,
+if the initial responders think this framework will be useful in taking
+care of the problem, they should invoke the incident response process.
+
+## Roles
+
+### Incident Commander
+
+The incident commander runs the incident response. This *does not* mean
+that they are leading the investigation; in fact, they should not be
+investigating at all directly. Their responsiblities are:
+
+* Keep track of the current state of the incident response; what is the
+  impact to the service and users? What is the state of any mitigations
+  or attempted fixes? How are the other members of the incident response
+  team holding up?
+* Make any final decisions about incident response. No one should put in
+  place any mitigations, attempted solutions, or active diagnostic
+  measures *without* running it by the IC first. This is imperative to
+  ensure no one is working at cross purposes and that the current state
+  of the system is understood.
+
+During especially severe incidents, the commander may also have a deputy
+to assist with their tasks.
+
+### Technical Lead/Investigator
+
+There may be one or more investigators on any one incident. These are the
+people responsible for actual diagnosis of problems and implementation of
+any mitigations or attempted fixes for the cause of the incident.
+
+If there is more than two investigators on the incident, one should be
+designated as the technical lead. This will be the person who reports to
+the incident commander and coordinates the investigation to keep people
+from duplicating effort or stepping on each other's toes.
+
+Before taking any *active* measures to investigate, the investigator(s)
+should coordinate with the incident commander to make sure they are aware
+and approve of taking the action.
+
+### Communicator
+
+The communicator is responsible for handling external communication during
+the incident, both by keeping the rest of the engineering team informed
+and notifying stakeholders (such as clients) about the status of the
+incident. Similarly, if non-responders want to ask questions or notify
+the incident response team of issues, they should use the communicator as
+a conduit to the incident commander, not talk to the commander directly.
+
+During less severe incidents, the communicator role may be folded into
```

Comment:
> During less severe incidents, the communicator role may be folded into
the scribe role.

I don't know that ever seen an actual communicator role in use. I typically see the IC handle most communications, especially with the customer.

Reply:
I have *definitely* seen a communicator role in use during big incidents; the IC should definitely not be handling communication with stakeholders and customers, which can often be a heavy lift. Having a separate person (either a dedicated communicator or the scribe) handle it allows them to focus on actually resolving the incident and not losing context, which should be top priority.

Reply:
+1 to separate communicator. Did that at my last job and it was such a nice experience to have someone on point for it specifically.